diff --git a/node_modules/react-native-country-codes-picker/index.tsx b/node_modules/react-native-country-codes-picker/index.tsx
index 28077f3..d4300d6 100644
--- a/node_modules/react-native-country-codes-picker/index.tsx
+++ b/node_modules/react-native-country-codes-picker/index.tsx
@@ -195,7 +195,7 @@ export const CountryPicker = ({
     };
 
     const renderItem = ({ item, index }: { item: CountryItem, index: number }) => {
-        let itemName = item?.name[lang];
+        let itemName = (item?.name['en'] === 'Anguilla' && lang === 'ar') ? 'أنغويلا' : item?.name[lang];
         let checkName = itemName?.length ? itemName : item?.name['en'];
 
         return (
